packages/legend-application-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -113,6 +116,24 @@ const checkForDeprecatedNode = (
   return false;
 };
 
+const checkForAlreadyUsedNode = (
+  node: QueryBuilderExplorerTreeNodeData,
+  projectionState: QueryBuilderProjectionState,
+): boolean => {
+  if (node.id === '@dummy_rootNode' && projectionState.columns.length) {
+    return true;
+  } else if (
+    projectionState.columns.some(
+      (column) =>
+        column instanceof QueryBuilderSimpleProjectionColumnState &&
```

Comment:
what should we do with QueryBuilderDerivationProjectionColumnState?

Reply:
It's hard to analysis for that so we decided not to do